Elaborate Neutral landing page and Tanner's role as solo-evil

TheHans255 opened the pull request:
# Description

- Add page for the Skinchanger
- Expand the neutral page to distinguish between "free agent" Neutrals and solo evils
- Elaborate the Tanner's role as a solo evil

# Checklist

I have checked the following _(mark as completed with [x])_:

- [x] The content is accurate (and confirmed with the mods)
- [x] Casing and wording of keywords such as role names, factions, etc, is all correct
- [x] The Markdown is valid
- [x] It renders correctly _(follow README for instructions on previewing)_
- [x] The sidebar links are correct and in the right sections _(follow README for instructions on previewing)_

Commit:
Distinguish between free agents and solo evils

- The Neutral umbrella contains two role types: free agents who
act amicably neutral and eventually join a faction,
and solo-evils who work against the entire village with their
own win condition. Expand the Neutral landing page to reflect this.

Commit:
Expand Tanner to reflect status as solo-evil

- Since the Tanner is attempting to work against the village
and win independently, they are best classed as a solo evil.
Change their page to reflect this.
- Also do cleanup of the text to reflect additional factions and
edge cases, including multiple Tanners.

Commit:
WIP: Add Skinchanger page

- Reflect information seen in existing Skinchanger games
- Add notes about common Skinchanger strategies
- TODO: Add dates for the timeline events

TheHans255 commented:
I am unfortunately not able to build on Windows - the build spirals out of control and runs out of memory.

Also, I need dates both for the Skinchanger's introduction and for its change to not require shifting alongside each kill.

ShaneMcC commented:
ext-342 for the introduction (https://archive.werewolv.es/#game=ext-342&phase=PreGame)
2020-05-10 for the shifting change

StormPooper commented on src/guides/neutral/skinchanger.mdx:
```diff
@@ -0,0 +1,41 @@
+## History
+
+<Timeline lineColor="white">
+  <Event interval="TODO">No longer required to shift into their kill target each night, but must shift at least once every 3 nights to survive</Event>
```
// TODO

TheHans255 replied:
Done

Commit:
Merge branch 'site' into thehans255/add-skinchanger

Commit:
Make necessary corrections to Skinchanger page

- Import Timeline and Event on both Skinchanger and Tanner
- Skinchanger is not WC
- Make Skinchanger's "it" pronouns consistent
- Add dates for Skinchanger changes
- Make "Stalker can follow you" text clearer

Kirschstein commented:
I've merged in Helix's branch which has a skin changer page, if you want some parts of yours amalgamated it'll have to be re-done, but I recall seeing you were happy with Helix's.

Commit:
Merge branch 'site' into thehans255/add-skinchanger

Commit:
Re-add React imports to skinchanger.mdx

TheHans255 commented:
> 
> 
> I've merged in Helix's branch which has a skin changer page, if you want some parts of yours amalgamated it'll have to be re-done, but I recall seeing you were happy with Helix's.

I just did a conflict resolution. I'm happy with Helix's skinchanger stuff, but I do want to keep my additions to the landing page and the Tanner page.

Kirschstein commented on src/guides/neutral/tanner.mdx:
```diff
@@ -4,9 +4,13 @@ menu: The Neutral
 # Tanner
 
-The Tanner is a neutral player. They hate their job and they want out. The Tanner wins the game by getting lynched. If they are lynched, the game immediately ends.
+The Tanner is a solo-evil neutral player. The Tanner hates their job, the village is forcing them to do it, and they want out.
```
```suggestion
The Tanner is a solo neutral player. The Tanner hates their job, the village is forcing them to do it, and they want out.
```

I'm not entirely sure where the line is between evil/neutral in the game, but the Tanner certainly feels "neutral" and not "evil" to me

StormPooper replied:
I'd agree, they don't really have an alignment other than their own desires.

StormPooper commented on src/guides/neutral/tanner.mdx:
```diff
@@ -17,13 +21,16 @@ The Tanner is a neutral player. They hate their job and they want out. The Tanne
 
-- The Tanner only wins if they are lynched, being killed at night does not count.
-- The Tanner cannot be recruited by sub-faction leaders (Mason, Inquisition, Illuminati). However, they can be recruited by a Vampire or Vampire Master.
 - The Tanner is only used in Speed Games.
```
```suggestion
- The Tanner is primarily used in Speed Games.
```

Kirschstein replied:
**only**

StormPooper replied:
Worth a shot ;)

Commit:
Solo instead of solo-evil

StormPooper commented:
Closing due to age and because of merge conflicts, feel free to reopen and fix if you'd still like to get these updates in.
